setup.py

```diff
@@ -3,17 +3,25 @@
 import os.path
 
 package_name = "dbt"
-package_version = "0.1.0-SNAPSHOT"
+package_version = "0.1.0"
 
 setup(
   name=package_name,
   version=package_version,
+  description="Data build tool for Analyst Collective",
+  author="Analyst Collective",
+  author_email="admin@analystcollective.org",
+  url="https://github.com/analyst-collective/dbt",
   packages=find_packages(),
+  entry_points={
+    'console_scripts': [
+      'dbt = dbt.main:main',
+    ],
+  },
   scripts=[
     'scripts/dbt',
   ],
   install_requires=[
-    'argparse>=1.2.1',
     'Jinja2>=2.8',
     'PyYAML>=3.11',
```

Comment:
I wrote this before that guy liberated his NPM modules haha. Yeah, good idea 

Sent from my iPhone

> On Mar 25, 2016, at 1:47 PM, SamProtas <notifications@github.com> wrote:
> 
> In setup.py:
> 
> >    scripts=[
> >      'scripts/dbt',
> >    ],
> >    install_requires=[
> > -    'argparse>=1.2.1',
> >      'Jinja2>=2.8',
> >      'PyYAML>=3.11',
> Should we version lock these to minimize the chance of breakage?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>